src/main/webapp/assets/modules/common/securityService.js

```diff
@@ -3,34 +3,32 @@ techlooper.factory("securityService", function (apiService, $rootScope, $q, util
   //localStorage.setItem('CAPTURE-PATHS', '/');
 
   var instance = {
-    //logout: function () {
-    //  apiService.logout()
-    //    .success(function (data, status, headers, config) {
-    //      $rootScope.userInfo = undefined;
-    //      $location.path("/");
-    //    });
-    //},
+    logout: function () {
+      apiService.logout()
+        .success(function (data, status, headers, config) {
```

Comment:
'config' is defined but never used.<br>'headers' is defined but never used.<br>'status' is defined but never used.<br>'data' is defined but never used.